data/STM32C011D6Yx.json

```diff
@@ -25,6 +25,22 @@
       "max": 85.0
     }
   },
+  "parts": [
+    {
+      "mpn": "STM32C011D6Y6TR",
+      "temperature_min": -40.0,
+      "temperature_max": 85.0,
+      "packing_type": "Tape and Reel",
+      "status": "Active"
+    },
+    {
+      "mpn": "STM32C011D6Y3TR",
+      "temperature_min": -40.0,
+      "temperature_max": 125.0,
+      "packing_type": "Tape and Reel",
+      "status": "Coming soon"
```

Comment:
We should probably ignore or re-map "Coming soon" parts. But that's a choice for the parts generator.

Reply:
That was exactly my intention! :smiley: In addition, for the non-skipped MPNs I'd also add this status information as an attribute to the parts because the status "NRND" is then directly visible in the component picker dialog.

But as you said, this is the decision of the parts generator.